flaml/automl/state.py

```diff
@@ -56,6 +57,7 @@ def __init__(
         self,
         learner_class,
         data_size,
+        data,
```

Comment:
A minor issue: When `data` is provided, `data_size` is presumably not necessary (could be induced from the given data).

Reply:
Fixed.